Gopkg.toml

```diff
@@ -0,0 +1,115 @@
+[[constraint]]
+  name = "github.com/awalterschulze/gographviz"
+  version = "1.0.0"
```

Comment:
Wondering if we want to switch to use specific revisions for all, to avoid someone pushing a malicious version to the repo? cc @Roasbeef @cfromknecht

Reply:
As in commit hashes everywhere? SGTM

Reply:
Yeeh. Would also avoid huge changes to `glide.lock` (if this concept applies to `dep`) when updating one dependency.